commands/roll.py

```diff
@@ -0,0 +1,5 @@
+async def roll(ctx):
+    """ Rolls a pseudorandom number from 0 to 100 """
```

Comment:
Lets update the comment syntax here:
```suggestion
    # Rolls a pseudorandom number from 0 to 100
```